### PR TITLE
Switch left and right

### LINKED ISSUE #20: Switch left and right in diff editor
If I could suggest to switch left and right. I you look at changes in git extension for example: local content is always on the right. I'm always confusing when comparing folder with your extension.

### PR DESCRIPTION
**Pull request by llgcode**
Fixes #20

### CODE REVIEW

File: src/extension.ts [hunk 1]
```diff
@@ -1,26 +1,20 @@
...
-export function activate() {
+
+export function activate(context: ExtensionContext) {
   const onlyInA = new ViewOnlyProvider();
   const onlyInB = new ViewOnlyProvider();
   const foldersCompareProvider = new CompareFoldersProvider(onlyInA, onlyInB);
-
-  disposables.push(
-    ...[
+  context.subscriptions.push(
```
Comment (moshfeu) [inline src/extension.ts]: 👍

**Comment by llgcode** [comment]
Can you suggest a name for the setting please? and some possible values...

**Comment by moshfeu** [comment]
Maybe "compare layout" 

values

- my <> compared
- compared <> my

**Comment by moshfeu** [comment]
@llgcode want to continue with it?
Thanks!

**Comment by llgcode** [comment]
Yes I will do ot when have time. I think next week

**Comment by moshfeu** [comment]
If I'll have time before, I'll complete it.

**Pull request merged**